src/components/ClientInfo/ClientInfo.tsx

```diff
@@ -298,17 +247,47 @@ const ClientActionsBox = ({cases}) => {
   // This function is triggered when the select changes
   const handleSelectCaseValue = (event: React.ChangeEvent<HTMLInputElement>) => {
     setSelectCaseValue(event.target.value);
+    const caseFromType = getCaseFromType(event.target.value);
+    setSelectedCaseInfo(caseFromType);
+    console.log(caseFromType);
   };
   const handleAccept = (event: React.ChangeEvent<HTMLInputElement>) => {
     if (selectCaseValue) {
       setClientActionsState("confirm approve");
     }
   };
+  // async function to update backend
+  const handleConfirm = async(status: string) => {
+    setUpdatingStatus(true);
+    const caseId = selectedCaseInfo.id;
+    console.log(selectedCaseInfo);
+    await updateStatus(client.id, caseId, status, selectedCaseInfo);
+    setSelectCaseValue(null);
+    setUpdatingStatus(false);
+  };
+  const getCaseFromType = (type: string): Case => {
+    const filteredCase = caseInfo.filter((c) => {
+      return (CaseKey[c.type] === type);
+    });
+    return filteredCase[0];
+  };
+  // renders correct message for selected case
   const caseSelectedComp = ():React.Component => {
```

Comment:
On the return values, I'm getting an error that Type 'Element' is not assignable to type 'Component<{}, {}, any>' (this also happens on displayDoc for me)